src/main/java/uk/gov/hmcts/reform/draftstore/endpoint/v3/DraftController.java

```diff
@@ -123,17 +107,9 @@ public DraftList readAll(
         @RequestHeader(SERVICE_HEADER) String serviceHeader,
         @RequestBody @Valid UpdateDraft updatedDraft
     ) {
-        UserAndService userAndService = authService.authenticate(authHeader, serviceHeader);
```

Comment:
To me this line makes it a little bit clearer, as it's not obvious what authService.authenticate returns. Just to be considered.